.github/workflows/linux.bash

```diff
@@ -1,12 +1,40 @@
 #!/bin/bash
 set -exo pipefail
 
-export PATH="/usr/lib/ccache:/usr/lib64/ccache:$PATH"
+export PATH="/usr/lib/ccache/bin:/usr/lib/ccache:/usr/lib64/ccache:$PATH"
 export CCACHE_DIR=/icinga2/ccache
 export CTEST_OUTPUT_ON_FAILURE=1
 CMAKE_OPTS=''
 
 case "$DISTRO" in
+  alpine:*)
+    # Install packages from Alpine package, just
+    # - LibreSSL instead of OpenSSL 3 and
+    # - no MariaDB or libpq as they depend on OpenSSL.
+    # https://gitlab.alpinelinux.org/alpine/aports/-/blob/master/community/icinga2/APKBUILD
+    apk add \
+      bison \
+      boost-dev \
+      ccache \
+      cmake \
+      flex \
+      g++ \
+      libedit-dev \
+      libressl-dev \
+      ninja-build \
+      samurai \
+      tzdata \
+      yajl-dev
```

Comment:
What's yajl-dev for?

Reply:
https://github.com/Icinga/icinga2/pull/9949/files/085fa713fd1f665e3a76be01656f19d814748646#r1909074867

Reply:
Then they haven't cleaned up their dependencies for quite a while. yajl was used before the code switched to using nlohmann::json (which happened ages ago).

Reply:
You guys clean your package build dependencies? /s